src/test/clojure/xtdb/sql_test.clj

```diff
@@ -1893,3 +1893,39 @@
   
   (t/is (= [{:xt/column-1 "public"}]
            (xt/q tu/*node* "SELECT current_schema FROM (VALUES 1) AS x"))))
+
+(t/deftest test-postgres-access-control-functions
+  ;; These current functions should always should return true
```

Comment:
Could dry up these test examples, so that the `expr` is templated into the query for each case. See `plan-expr` for inspiration, or change these to plan tests and use it directly.